Lesson 4-1

vitaliipp opened the pull request:
The Project Structure has been improved
Component Prop Types have been assigned

vitaliipp committed:
Improve project structure

vitaliipp committed:
assign component prop types

Jkuffler commented:
Overall this has been done quite well; though, there is one warning displaying for your app...

_Warning: Failed prop type: The prop `children` is marked as required in `InputWithLabel`, but its value is `undefined`._

There are a couple of ways you might approach a resolution here. If you would like assistance from colleagues or mentors, then please use the general discussion in Slack 👍 Please correct prior to next submission - tyvm

vitaliipp committed:
fix warning

vitaliipp commented:
Thank you for the review. I have fixed a warning in the code. In the current version of my project, I am not utilizing the 'children' prop. However, this prop is a requirement for the final project, so I have added an empty string as the 'children' prop.
